ComposeSample/app/src/main/java/com/microsoft/device/display/samples/composesample/HomePage.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.composesample
+
+import android.util.Log
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.ScrollableColumn
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.preferredHeight
+import androidx.compose.foundation.layout.preferredWidth
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.ui.tooling.preview.Preview
+import com.microsoft.device.display.samples.composesample.models.DataProvider
+import com.microsoft.device.display.samples.composesample.models.ImageModel
+import com.microsoft.device.display.samples.composesample.viewModels.AppStateViewModel
+
+private lateinit var appStateViewModel: AppStateViewModel
+
+@Preview
+@Composable
+fun HomePreview() {
+    val models = DataProvider.imageModels
+    ShowList(models = models)
+}
+
+@Composable
+fun Home(viewModel: AppStateViewModel) {
+    appStateViewModel = viewModel
+    SetupUI()
+}
+
+@Composable
+fun SetupUI() {
+    val models = DataProvider.imageModels
+    val isScreenSpannedLiveData = appStateViewModel.getIsScreenSpannedLiveData()
+    val isScreenSpanned = isScreenSpannedLiveData.observeAsState(initial = false).value
+
+    Log.i("ComposeSample", "SetupUI isScreenSpanned: $isScreenSpanned")
```

Comment:
You can add a constant like `TAG` with the name you want so you avoid to write always the identifier in your logs.